src/deluge/gui/ui/sound_editor.cpp

```diff
@@ -816,6 +838,16 @@ ActionResult SoundEditor::potentialShortcutPadAction(int32_t x, int32_t y, bool
 			goto doSetup;
 		}
 
+		//For Kit Instrument Clip with Affect Entire Enabled
```

Comment:
hmm I'm a bit less sure about this, I wonder if this will lead to more confusion than it helps. Maybe these should only work if you've already opened the kit fx menu?

Reply:
Agreed. Made the change as suggested :)